### PR TITLE
Add missing load_default_certs() call.

### PR DESCRIPTION
**Pull request by moisesguimaraes**
Fixes: #349

### CODE REVIEW

**Comment by matusvalo** [comment]
Thank you @moisesguimaraes for your PR. Can you also adjust integration tests with case covering this PR? Now they are not failing on master:
```
$ pytest -sv -E rabbitmq t/integration/test_rmq.py
Test session starts (platform: linux, Python 3.8.5, pytest 5.3.5, pytest-sugar 0.9.4)
cachedir: .pytest_cache
rootdir: /home/matus/dev/py-amqp, inifile: setup.cfg
plugins: rerunfailures-9.1.1, forked-1.3.0, sugar-0.9.4, xdist-2.1.0, cov-2.10.1, travis-fold-1.3.0, case-1.5.3
collecting ...
 t/integration/test_rmq.py::test_connect[plain] ✓                                                                                                                                                   5% ▌
 t/integration/test_rmq.py::test_connect[tls] ✓                                                                                                                                                    10% ▉
 t/integration/test_rmq.py::test_tls_connect_fails ✓                                                                                                                                               14% █▌
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish-False-True] ✓                                                                                        19% █▉
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish-True-True] ✓                                                                                         24% ██▍
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish-False-False] ✓                                                                                       29% ██▉
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish-True-False] ✓                                                                                        33% ███▍
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish_confirm-False-True] ✓                                                                                38% ███▊
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish_confirm-True-True] ✓                                                                                 43% ████▍
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish_confirm-False-False] ✓                                                                               48% ████▊
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[plain-basic_publish_confirm-True-False] ✓                                                                                52% █████▎
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish-False-True] ✓                                                                                          57% █████▊
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish-True-True] ✓                                                                                           62% ██████▎
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish-False-False] ✓                                                                                         67% ██████▋
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish-True-False] ✓                                                                                          71% ███████▎
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish_confirm-False-True] ✓                                                                                  76% ███████▋
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish_confirm-True-True] ✓                                                                                   81% ████████▏
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish_confirm-False-False] ✓                                                                                 86% ████████▋
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_consume[tls-basic_publish_confirm-True-False] ✓                                                                                  90% █████████▏
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_get[plain] ✓                                                                                                                     95% █████████▌
 t/integration/test_rmq.py::test_rabbitmq_operations.test_publish_get[tls] ✓                                                                                                                      100% ██████████

Results (0.54s):
      21 passed
```

File: t/unit/test_transport.py [hunk 1]
```diff
@@ -639,112 +640,144 @@ def test_wrap_context(self):
...
-            suppress_ragged_eofs=True,
-            server_hostname=None
-        )
-        assert ret == sentinel.WRAPPED_SOCKET
+            context.load_cert_chain.assert_not_called()
+            context.load_verify_locations.assert_not_called()
+            context.set_ciphers.assert_not_called()
+            context.verify_mode.assert_not_called()
+
+            context.load_default_certs.assert_called_with(
```
Comment (moisesguimaraes) [inline t/unit/test_transport.py:654]: @matusvalo I have tests to cover the new code path already.

Comment (matusvalo) [reply t/unit/test_transport.py:654]: yes I saw. But I was curious why integration test creating real tls connection to broker suit does not fail on master.

**Comment by moisesguimaraes** [comment]
Ok, added some integration tests.

File: t/unit/test_transport.py (hunk 1 above)
Comment (moisesguimaraes) [reply t/unit/test_transport.py:654]: They should only fail when you try to load system certs. I added new tests for this case.

**Comment by auvipy** [comment]
> Ok, added some integration tests.

Our CI is not working, so please check on you repo if they are working fine.

**Review by auvipy — COMMENTED** [review]
looks good to me but waiting for confirmation

**Comment by matusvalo** [comment]
I will check it today.

**Comment by matusvalo** [comment]
OK I have checked the PR and it looks great! Thank you @moisesguimaraes ! I also like the integration test. Thanks again!

**Comment by auvipy** [comment]
i will try to release tonite or tomorrw morning

**Pull request merged**